autoswitch_virtualenv.plugin.zsh

```diff
@@ -167,8 +167,8 @@ function rmvenv()
         fi
 
         printf "Removing ${PURPLE}%s${NORMAL}...\n" "$venv_name"
-        rm -rf "$(_virtual_env_dir "$venv_name")"
-        rm ".venv"
+        /bin/rm -rf "$(_virtual_env_dir "$venv_name")"
+        /bin/rm ".venv"
```

Comment:
could you put a comment here explaining why we use absolute paths?

I wonder if this would break for anyone. I think it's safe to assume this is the path on all distros but if we find a bug the a workaround would be `$(=rm)`

Reply:
Sure. I'll add a comment. . I don't think it would break as the Linux standard specifies it https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s04.html (although https://fedoraproject.org/wiki/Features/UsrMove uses a symbolic link to /bin).

Reply:
Nice find! Could you also put the link in a comment as a reference to why we are confident this will work?

Reply:
Done